### PR TITLE
avoid frame skips

### PR DESCRIPTION
**Pull request by tgerdesnv**
Some parts of the func stack were avoided to try to keep the name clean and short.  This PR adds many of them back in.

Don't merge this until a twin story can be pulled in in DLProf.

### CODE REVIEW

**Review by debermudez — APPROVED** [review]

**Comment by debermudez** [comment]
I'll merge once CI passes

**Comment by tgerdesnv** [comment]
There is a mirror story on dlprof to post process. I don't have the MR up yet.  don't merge yet.
________________________________

**Pull request merged**
